spec/migration_spec.cr

```diff
@@ -0,0 +1,29 @@
+require "./spec_helper"
+
+class MigrationThatPartiallyWorks::V999 < LuckyMigrator::Migration::V1
+  def migrate
+    create :fake_things do
+      add foo : String
+    end
+
+    alter :table_does_not_exist do
+      add foo : String
+    end
+  end
+
+  def rollback
+  end
+end
+
+describe LuckyMigrator::Migration::V1 do
+  it "executes statements in a transaction" do
```

Comment:
This works well, but it requires the database to be created first. I tried creating the db manually in a context block, but it printed to the console when it already existed.

What is the correct way to handle this?

Reply:
I believe if you use “Lucky db.create” it should have a database that it can connect to. If that’s the case then I think that’s good.ill just need to update the readme to inform people they need to do that if they’d like to contribute

Reply:
Yeah it works if you create the db and migrate it so it has the `migrations` table, so a message in the Readme like in `lucky_record` should work.